modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Controller/PublisherMethodsController.cs

```diff
@@ -52,11 +52,6 @@ public class PublisherMethodsController : IMethodController {
         /// </summary>
         public async Task<PublishedNodesResponseApiModel> UnpublishAllNodesAsync(
             PublishNodesEndpointApiModel request) {
-
-            if (request.OpcNodes != null) {
```

Comment:
Let's keep this check as we should not have any `OpcNodes` in the request. I would actually change the check to `if (request.OpcNodes != null && request.OpcNodes.Count > 0)` just to allow for empty lists as well.